SU-HSDO/suhumsci
# ARCH-155 mission statement and emphasized text styles

# READY FOR REVIEW

# Summary
- Added emphsized-text style to WYSIWYG editor and added related styles
- Added custom homepage styles for mission statement for Archaeology's homepage

# Needed By (Date)
- End of sprint (8/2)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Add emphasized-text style to collection of text on homepage's mission statement paragraph
3. Ensure that the mission statement closely resembles the mockup at https://invis.io/P7HCI9HYNA4#/299456614_Homepage

# Affected Projects or Products
- Archeaology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-155

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
